app/Http/Controllers/ModuleController.php

```diff
@@ -9,11 +9,24 @@
 class ModuleController extends Controller
 {
 
-    public function moduleStatistics(int $courseId): SuccessResponse
+    public function moduleStatistics(Request $request, int $courseId): SuccessResponse
     {
         $moduleService = new ModuleService();
```

Comment:
Dependency Injection: The ModuleService class is instantiated directly in the controller method, which can lead to tight coupling and make the code difficult to test. It would be better to inject the ModuleService class as a dependency in the constructor of the controller and use dependency injection to pass it to the controller method.